src/spec-configuration/containerCollectionsOCI.ts

```diff
@@ -304,7 +304,10 @@ async function generateScopeTokenCredential(output: Log, registry: string, ociRe
 
 	const authServer = registry === 'docker.io' ? 'auth.docker.io' : registry;
 	const registryServer = registry === 'docker.io' ? 'registry.docker.io' : registry;
-	const url = `https://${authServer}/token?scope=repository:${ociRepoPath}:${operationScopes}&service=${registryServer}`;
+
+	// TODO: Probably add other registries here, but I'm not sure which ones.
+	const isOauth2 = registry.endsWith('azurecr.io');
```

Comment:
For Azure Container Registry, azurecr.io (public) / azurecr.cn (China) / azurecr.us (US Gov)

Reply:
Done.